dali/operators/reader/reader_op.h

```diff
@@ -80,7 +81,8 @@ class DataReader : public Operator<Backend> {
   // perform the prefetching operation
   virtual void Prefetch() {
     // We actually prepare the next batch
-    TimeRange tr("DataReader::Prefetch #" + to_string(curr_batch_producer_), TimeRange::kRed);
+    DomainTimeRange tr("[DALI] DataReader::Prefetch #" + to_string(curr_batch_producer_),
```

Comment:
Can we make them
```suggestion
    DomainTimeRange tr("[DALI][DataReader] Prefetch #" + to_string(curr_batch_producer_),
```
and so on?

Reply:
Done